src/pages/project/cartzoom.js

```diff
@@ -104,5 +107,40 @@ module.exports = {
     matrix[0] = zoom;
     matrix[3] = zoom;
     this.setState({matrix});
+  },
+
+  // Get the bounding rect for the currently visible page area
+  getPageBoundingRect: function () {
+    // Calculate how the current page is situated in the map container
+    var scale = this.getMaxPageSize();
+    var tileWidth = this.cartesian.width * scale;
+    var tileHeight = this.cartesian.height * scale;
+    var elMap = document.querySelector('#map');
```

Comment:
i think actually you can use `getClientBoundingRect` here since it does take into account transforms instead